pawsitive/app/views/users/profile.html.erb

```diff
@@ -1,3 +1,5 @@
 <% if current_user.role == 'member' %>
-  <%= render 'users/member_profile' %>
+    <%= render 'users/member_profile' %>
+  <% else %>
+    <%= render 'users/users_profile' %>
```

Comment:
the actual profile page/template is missing.

Reply:
I just pushed changes can you try again